src/constants/button.js

```diff
@@ -61,3 +61,7 @@ export const MENU_PLACEMENT = {
     ABOVE:   ('above' : 'above'),
     BELOW: ('below' : 'below')
 };
+
+export const EXPERIENCE = {
```

Comment:
I like the new `EXPERIENCE` addition. It will scale with supporting more possible values in the future 💯